internal/controller/driver_controller.go

```diff
@@ -921,6 +923,20 @@ func (r *driverReconcile) reconcileControllerPluginDeployment() error {
 	return err
 }
 
+func (r *driverReconcile) ControllerPluginCsiAddonsContainerPort() corev1.ContainerPort {
```

Comment:
```suggestion
func (r *driverReconcile) controllerPluginCsiAddonsContainerPort() corev1.ContainerPort {
```

Lets not export this function

Reply:
I had applied it previously and had to essentially undo it to fix the build.